server/urls.py

```diff
@@ -39,6 +24,7 @@
     path('issue/', issue),
     path('docs/', schema_view.with_ui('swagger',cache_timeout=0), name = 'schema-swagger-ui'),
     path('admin/', admin.site.urls),
+    path('logout/', auth_views.LogoutView.as_view(), name='logout'),
```

Comment:
Can I test this? 😅

Reply:
I'd like to believe that 😂

Reply:
On the right-hand side, you may see this. When you logout, it disappears ✨  

![image](https://user-images.githubusercontent.com/49693160/106452575-1b4c3b80-64ae-11eb-8abc-e4c8a6af3602.png)

PS- This might seem unwanted and will be removed if you force me to do so 😂

Reply:
Got it, and I won't ask you to remove it 😂